deep-security-agent/CHANGELOG.md

```diff
@@ -1,43 +1,17 @@
-7072714t(2 weeks ago): Added a block to ensure that the sleep() delay is executed in sequence on all OSes
-82d8e86t(3 weeks ago): Adjusted package method for SUSE clients
-b2e3ba8t(3 weeks ago): Added extra logging to provide better information in the Chef logs
-1b092bbt(3 weeks ago): Adjusted sleep time to 15s to match current recommendations in 'deployment scripts' in Deep Security
-ce43935t(5 weeks ago): Updated README to reflect new OpsWorks structure
-4acac86t(6 months ago): Updated README
-0e18f8at(6 months ago): Updated documentation for recipe
-73bbe7dt(7 months ago): Updated README
-ad65e72t(7 months ago): Updated the format in the recipes README
-c8fe87bt(7 months ago): Updated the format in the recipes README
-66de374t(7 months ago): Updated the format in the recipes README
-44d9cdct(7 months ago): Updated the recipes README
-5184508t(7 months ago): Added a README for the recipes folder
-a05936ct(7 months ago): Added instructions on how to merge multiple cookbooks for OpsWorks usage
-9e33d21t(7 months ago): Added instructions on how to merge multiple cookbooks for OpsWorks usage
-b146b52t(7 months ago): Added instructions on how to merge multiple cookbooks for OpsWorks usage
-5c68b21t(7 months ago): Updated main README to highlight AWS OpsWorks usage
-9ff14bdt(7 months ago): Updated main README to highlight AWS OpsWorks usage
-0753dd8t(7 months ago): Updated CHANGELOG with a note about the latest commit ID
-0a2acaet(7 months ago): Updated CHANGELOG with a note about the latest commit ID
-5eec530t(7 months ago): Added a title to the CHANGELOG
-e47a3d9t(7 months ago): Working on the format of 'git log' to ensure the CHANGELOG is up to date
-b70f02ft(7 months ago): Updated CHANGELOG with git commit history
-ed44ec4t(7 months ago): Updated the ## Usage instructions in the README
-a775a23t(7 months ago): Updated the README for clarity
-60f0918t(7 months ago): Updated attribute listing in the README
-08fe208t(7 months ago): Updated directory structure for OpsWorks compatibility as per http://docs.aws.amazon.com/opsworks/latest/userguide/workingcookbook-installingcustom-repo.html
-3522484t(7 months ago): Moved the deploy-dsa.rb recipe to default.rb
-eb542fet(7 months ago): Added more recipes to manage the agent on the node
-d6822c0t(7 months ago): Added more recipes to manage the agent on the node
-e836b72t(7 months ago): Added a recipe to force an agent heartbeat/check-in
-fea5c69t(7 months ago): Updated README
-baaf4dat(7 months ago): Updated README
-8e8c5c8t(7 months ago): Update README with the required attributes for the deploy-dsa.rb recipe
-eda4a3et(7 months ago): Removed port from dsm_agent_download_hostname, it's in it's own attribute now
-960935bt(7 months ago): Added initial test recipe for deploying the Deep Security agent
-11118e7t(7 months ago): Added a default cookbook
-cd07e1bt(7 months ago): Updated README
+# Change Log
+
+## [1.1.0](https://github.com/deep-security/chef/tree/1.1.0) (2018-01-17)
```

Comment:
As the change log will not contain the release itself, the in consist may cause confusing.
I suggest to show the release page link on github directly
https://github.com/deep-security/chef/releases